src/Conversions.cc

```diff
@@ -61,12 +61,14 @@
 #include <string>
 
 #include "ignition/gazebo/Conversions.hh"
+#include "ignition/gazebo/Export.hh"
 #include "ignition/gazebo/Util.hh"
 
 using namespace ignition;
 
 //////////////////////////////////////////////////
 template<>
+IGNITION_GAZEBO_VISIBLE
```

Comment:
Just curious, is this really needed in the CC files?

Reply:
How the code is right now, it is needed. There is an external library (network) using the function in `NetworkManagerPrimary.cc`. Once compiled in ignition-gazebo5.lib, the symbols need to exported to be linked in a different object file.  Otherwise:

```
NetworkManagerPrimary.obj : error LNK2001: unresolved external symbol "class ignition::msgs::WorldStatistics __cdecl ignition::gazebo::v5::convert<class ignition::msgs::WorldStatistics>
```